src/libs/convertToLTRForComposer/index.android.ts

```diff
@@ -1,8 +1,53 @@
+import CONST from '@src/CONST';
 import ConvertToLTRForComposer from './types';
 
+/**
+ * Android only - We need to determinate when composer is considered empty this going to be used when we don't want to convert the input composer to LTR
+ */
+function hasBeenComposerConsideredEmpty(text: string): boolean {
+    // Regular expressions with all character that are consider as empty (spaces, unicode with spaces and unicode with at)
+    const emptyExpressions = [/^\s*$/, new RegExp(`^${CONST.UNICODE.LTR}@$`), new RegExp(`${CONST.UNICODE.LTR}\\s*$`)];
+    return emptyExpressions.some((exp) => exp.test(text));
+}
+
+/**
+ * Android only - We should remove the LTR unicode when the input is empty to prevent: Sending an empty message, metion suggestions not works if @ or \s (at or space) is the first character; (force option: always remove the unicode, going to be used when composer is consider as empty) or placeholder not shows if unicode character is the only remaining character. */
```

Comment:
Shorten this comment so it is less than 190 characters wide, please.

Reply:
My bad, fixed!